### PR TITLE
Viet tag detail activity

### PR DESCRIPTION
**Pull request by viethphamvn**

### CODE REVIEW

File: app/src/main/java/edu/temple/templetag/TagDetailActivity.java [hunk 1]
```diff
@@ -1,14 +1,100 @@
...
+        tagImageView = findViewById(R.id.tag_image_view);
+
+        tagLocationName.setText(mTag.getmTagLocationName());
+        tagCreatedBy.setText("created by " + mTag.getmTagCreatedBy());
+        tagUpVote.setText(mTag.getmTagUpvoteCount() + " Upvotes");
+        tagDownVote.setText(mTag.getmTagDownvoteCount() + " Downvotes");
+        tagPop.setText(mTag.getmTagPopularity() + " people are talking about this event");
+        tagDesc.setText(mTag.getmTagDescription());
+
+        closeBtn = findViewById(R.id.btn_delete);
```
Comment (dilloncoffman) [inline app/src/main/java/edu/temple/templetag/TagDetailActivity.java]: Should this be called deleteBtn?

**Review by dilloncoffman — CHANGES_REQUESTED** [review]
Should closeBtn be deleteBtn in TagDetailActivity?

File: app/src/main/java/edu/temple/templetag/TagDetailActivity.java (hunk 1 above)
Comment (viethphamvn) [reply app/src/main/java/edu/temple/templetag/TagDetailActivity.java]: lol yes. But I followed the UML Diagram

**Comment by viethphamvn** [comment]
Changed Botton Name

**Review by dilloncoffman — APPROVED** [review]

**Pull request merged**